### DISCUSSION TITLE
Lawrence O'Donnell MSNBC videos not detected

### DISCUSSION THREAD

**Discussion by FarneyBlather** [aclap-dev/video-downloadhelper#1372] [Help and Q&A]
### Your question

U.S.
YouTube

Recently, I have come across two MSNBC videos that Video Downloadhelper does not detect playing in YouTube. Both are short Lawrence O'Donnell spots. I discounted the first time it happened as a fluke, because the application works so well, otherwise. This time, I want to share my concern that something is affecting the ability to detect YouTube videos.

https://www.youtube.com/watch?v=88_Io_NOwuY

That is the video. Video Downloadhelper does not detect it. 

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"4a6b7781ef93409f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "E:\\Videos",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 1,
    "successfull_dl": 70,
    "history_limit_in_days": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 24H2 build 26100.2033, Firefox 131.0.3 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

Your MSNBC video downloaded in VDH without incident for me, & it played fine in VLC.  I didn't sit & watch it, just skimmed it to make sure it had audio & video at each sample point, which it did, right to the end.

You should search this forum for discussions of antivirus.  Make sure that's not a factor in your case.

**Comment by linx1337** [comment]
I am very pleased to hear that. The world would be a much better place if everyone couldn't watch MSNBC.
